checker/component_change.go

```diff
@@ -15,8 +16,9 @@ type ComponentChange struct {
 	Source  string `json:"source,omitempty" yaml:"source,omitempty"`
 }
 
-func (ComponentChange) MatchIgnore(ignorePath, ignoreLine string) bool {
-	return false
+func (c ComponentChange) MatchIgnore(ignorePath, ignoreLine string) bool {
+	return strings.Contains(ignoreLine, strings.ToLower(GetUncolorizedText(c))) &&
+		strings.Contains(ignoreLine, "components")
```

Comment:
I suggest to require the keyword "components" to ignore a component.

Reply:
I don't understand what you mean by "based on the json output it wouldn't work".
Can you explain the use case in more detail?